package.json

```diff
@@ -19,15 +19,19 @@
     "@hapi/boom": "^10.0.1",
     "@koa/cors": "^4.0.0",
     "@parse/node-apn": "^5.1.3",
+    "@types/node-fetch": "^2.6.2",
```

Comment:
Types should be in `devDependencies {...}`, use `yarn install -D ...` to put stuff in there